src/components/ResponsiveImage.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import { Box, Image } from 'grommet'
+import styled, { css } from 'styled-components'
+import PropTypes from 'prop-types'
+
+const StyledBox = styled(Box)`
+  display: inline;
+`
```

Comment:
If this needs to be display inline, just use a div. `Box` is default a div with flex styling, so you're removing the flex styling and any utility many of its props would have by changing the display.